src/restic/backend/b2/b2_test.go

```diff
@@ -0,0 +1,60 @@
+package b2_test
+
+import (
+	"os"
+	"restic"
+
+	"restic/errors"
+
+	"restic/backend/b2"
+	"restic/backend/test"
+)
+
+//go:generate go run ../test/generate_backend_tests.go
+
+func init() {
+	if os.Getenv("B2_ACCOUNT_ID") == "" || os.Getenv("B2_ACCOUNT_KEY") == "" {
+		SkipMessage = "No B2 credentials found. Skipping B2 backend tests."
+		return
+	}
+
+	cfg := b2.Config{
+		AccountID: os.Getenv("B2_ACCOUNT_ID"),
+		Key:       os.Getenv("B2_ACCOUNT_KEY"),
+		Bucket:    "restic-test",
```

Comment:
Bucket names need to be globally unique, and there's already a "restic-test" out there.  You should probably append a random string to this.